purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -4447,6 +4449,33 @@ class PurchasesTest {
         }
     }
 
+//    private fun mockOfferingsParser() {
+//        with(mockOfferingParser) {
+//            every {
+//                retrieveCustomerInfo(any(), any(), false, any())
+//            } answers {
+//                val callback = arg<ReceiveCustomerInfoCallback?>(3)
+//                if (errorGettingCustomerInfo == null) {
+//                    callback?.onReceived(mockInfo)
+//                } else {
+//                    callback?.onError(errorGettingCustomerInfo)
+//                }
+//            }
+//            every {
+//                cacheCustomerInfo(any())
+//            } just runs
+//            every {
+//                sendUpdatedCustomerInfoToDelegateIfChanged(any())
+//            } just runs
+//            every {
+//                updatedCustomerInfoListener = any()
+//            } just runs
+//            every {
+//                updatedCustomerInfoListener
+//            } returns null
+//        }
+//    }
+
```

Comment:
Do these need to be uncommented or rewritten yet?

Reply:
was gonna wait until amazonstoreproduct is done!